doc/doc-site/docs/off-chain/dss/changelog.md

```diff
@@ -13,11 +13,18 @@ Stable DSS builds are tracked on the [`dss-stable`] branch with tags like [`dss-
 1. Merge `main` into `dss-stable`.
 1. Push annotated tag to head of `dss-stable`.
 
-## [v2.1.0] (in progress)
+## [v2.1.0-rc1]
 
 ### Added
 
 - Price level events over MQTT (disabled by default, enable by adding `MQTT_PRICE_LEVELS=yes`, see `src/docker/example.env`) ([#753]).
+- Fix the `/tickers` `base_volume_nominal` field ([#749]).
```

Comment:
And `quote_volume_nominal`

Reply:
Also, the cited PR is wrong

#749 is for the schema permissions

#746 is for the tickers endpoint

Reply:
- [ ] Add link to #746 to bottom of changelog

Reply:
I would go against this as fixing permissions is not something that changed since the last version. I would just add the other PR here.